test/extract_slides.spec.js

```diff
@@ -354,4 +354,19 @@ describe('extractSlides', function() {
             return expect(slides).to.have.deep.property('[0].bodies[0].textRuns[0].baselineOffset', 'SUPERSCRIPT');
         });
     });
+
+    describe('with speaker notes', function() {
+        const markdown =
+            '# Title\n' +
+            '<!-- Hello **world** -->\n';
```

Comment:
Should you also include a test / expand this test to covert line breaks? Specifically wondering about how list items / indented code blocks are handled.